app/src/main/java/org/oppia/android/app/options/OptionControlsViewModel.kt

```diff
@@ -104,21 +169,35 @@ class OptionControlsViewModel @Inject constructor(
 
     optionsReadingTextSizeViewModel.readingTextSize.set(profile.readingTextSize)
 
-    itemViewModelList.add(optionsReadingTextSizeViewModel as OptionsItemViewModel)
-
-    if (enableLanguageSelectionUi.value) {
-      itemViewModelList.add(optionsAppLanguageViewModel as OptionsItemViewModel)
-    }
-
-    itemViewModelList.add(optionAudioViewViewModel as OptionsItemViewModel)
+    itemsList.add(optionsReadingTextSizeViewModel as OptionsItemViewModel)
+    itemsList.add(optionAudioViewViewModel as OptionsItemViewModel)
 
     // Loading the initial options in the sub-options container
     if (isMultipane.get()!! && isFirstOpen) {
       optionsReadingTextSizeViewModel.loadReadingTextSizeFragment()
       isFirstOpen = false
     }
 
-    return itemViewModelList
+    return itemsList
+  }
+
+  private fun processLanguageList(
```

Comment:
Why do we need 2 livedatas, 2 process methods, and a combine here? We should be removing the old functionality entirely since it's being replaced.

Reply:
HI Ben have removed one unnessesary livedata, i use the combine to merge data from the two different livedatas, what part exactly do you want changed on this.

Reply:
have updated largely. The combine function works on merging the LiveData objects.